source/_integrations/freebox.markdown

```diff
@@ -94,6 +95,10 @@ refreshes the devices states.
 This platform offers you sensors to monitor a Freebox router. The monitored conditions are
 instant upload and download rates in KB/s.
 
+## Service
+
+A service is available to reboot your Freebox router.
```

Comment:
Describe the service call. For a reference, check some other page on how they have added it.

Reply:
Is that better ?

Reply:
Yep, thanks!